### ISSUE TITLE
Getting 404 in most recent version of jupyterlab

### ISSUE THREAD

**Issue by ejolly** [ryantam626/jupyterlab_code_formatter#57]
Thanks for the great extension, but I'm having issues getting it to work on the latest version of jupyter lab (1.1.4) on macOS. I'm using the black code formatter with python (everything is installed in the same conda environment). I know that black works, because 1) pip can find it with `pip list | grep black` and 2) formatting code in my editor (VSCode) with black works just fine.

I see the the format notebook button at the top of my notebook in jupyterlab, and right-clicking on a cell shows the format cell option the dialog, but there are no commands listed in the command palette. Also in the server terminal I keep getting `404 POST /jupyterlab_code_formatter/format?1571308405340 (::1) 1.51ms referer=http://localhost:8888/lab` if I ever try to select either format option. 

Any recommendations?

**Comment by ryantam626** [comment]
Could you show me output of 
1. `pip freeze | grep 'jupyterlab\|rpy'`
2. `jupyter labextension list`
3. `jupyter serverextension list`
plase?

**Comment by ejolly** [comment]
Sure: 

![image](https://user-images.githubusercontent.com/8481982/67003146-5ce38800-f0ab-11e9-940b-9a08d745ca6b.png)

**Comment by ryantam626** [comment]
I think you are missing the server extension.

`pip install jupyterlab_code_formatter`
`jupyter serverextension enable --py jupyterlab_code_formatter`

**Comment by ejolly** [comment]
Ah weird that solved it. Sorry I thought installing the extension through jupyter lab discovery would include everything needed!

**Comment by swyu0711** [comment]
I execute all the commands and the jupyterlab_code_formatter server extentions is enabled, but I still got 404 POST error.

**Comment by ryantam626** [comment]
Try upgrading the version of the plugin, 1.0.2 should work, there was a
small issue with 1.0.1

**Comment by blackpill** [comment]
I also got 404 error.
jupyterlab_code_formatter version is 1.0.3
and jupyterlab version is 1.2.3

![image](https://user-images.githubusercontent.com/1231245/70524356-eed5b180-1b7f-11ea-8f5c-0c1d52334b0f.png)
![image](https://user-images.githubusercontent.com/1231245/70524433-16c51500-1b80-11ea-9eac-99b718675c24.png)

**Comment by ttesileanu** [comment]
I'm seeing this issue too: the code formatter doesn't work, the `http://localhost:8888/jupyterlab_code_formatter` page returns a jupyter-themed 404 error, while `http://localhost:8888/jupyterlab_code_formatter/formatters` returns 422 error.

Here are the outputs you asked for:
```
>>> pip freeze | grep 'jupyterlab\|rpy'
jupyterlab==1.2.6
jupyterlab-code-formatter==1.1.0
jupyterlab-server==1.0.6
```
```
>>> jupyter labextension list
JupyterLab v1.2.6
Known labextensions:
   app dir: /Users/ttesileanu/.pyenv/versions/3.7.4/envs/jupyter3/share/jupyter/lab
        @jupyter-widgets/jupyterlab-manager v1.1.0  enabled  OK
        @jupyterlab/debugger v0.1.2  enabled  OK
        @ryantam626/jupyterlab_code_formatter v1.1.0  enabled  OK
        jupyterlab-jupytext v1.1.1  enabled  OK
        jupyterlab-plotly v1.5.0  enabled  OK
        plotlywidget v1.5.0  enabled  OK
```
```
>>>  jupyter serverextension list
config dir: /Users/ttesileanu/.jupyter
    jupyterlab_code_formatter  enabled
    - Validating...
      jupyterlab_code_formatter  OK
config dir: /Users/ttesileanu/.pyenv/versions/3.7.4/envs/jupyter3/etc/jupyter
    jupyterlab  enabled
    - Validating...
      jupyterlab 1.2.6 OK
    jupytext  enabled
    - Validating...
      jupytext 1.3.3 OK
```

**Comment by ryantam626** [comment]
> http://localhost:8888/jupyterlab_code_formatter/formatters returns 422 error.
422 is expected because you didn't pass in a header to specify which plugin version are you on.

when you say "the code formatter doesn't work", what exactly did you try?

**Comment by ttesileanu** [comment]
Nothing showed up in the command palette and the shortcut I'd configured to apply Black formatting did nothing.

It mysteriously restarted working, though. I messed with a `json` configuration for a different extension in `.pyenv/versions/3.7.4/envs/jupyter3/etc/jupyter/jupyter_server_config.d` and added

```
{
  "jupyterlab_code_formatter": true
}
```

then restarted `jupyter-lab` and it started working, so I thought that somehow fixed it. But now I removed it and restarted `jupyter-lab` again, and it's still working, so I don't get it...
